### PR TITLE
Fix/[ios] BleError  handling capabilities

### PR DESCRIPTION
**Pull request by TomoLV**

### CODE REVIEW

File: ios/Classes/Event/ScanningStreamHandler.m [hunk 1]
```diff
@@ -27,7 +27,7 @@ - (void)onScanResult:(NSArray *)scanResult {
             if (scanResult[0] == [NSNull null]) {
                 scanResultsSink([JSONStringifier jsonStringFromJSONObject:scanResult[1]]);
             } else {
-                scanResultsSink([FlutterErrorFactory flutterErrorFromJSONString:scanResult[0]]);
+                scanResultsSink([FlutterErrorFactory flutterErrorForJSONString:scanResult[0]]);
```
Comment (mikolak) [inline ios/Classes/Event/ScanningStreamHandler.m]: ?

Comment (TomoLV) [reply ios/Classes/Event/ScanningStreamHandler.m]: Method was renamed in FlutterErrorFactory so it had to be changed here.

Comment (TomoLV) [reply ios/Classes/Event/ScanningStreamHandler.m]: There was also `flutterErrorForBleError` method added so change from `from` to `for` in the method name somehow sounded better for me.

Comment (mikolak) [reply ios/Classes/Event/ScanningStreamHandler.m]: But we want to create a Flutter-understandable error _from_ an instance of BleError, right?

**Comment by mikolak** [comment]
In conversation with @TomoLV it turned out these changes are unnecessary.

**Pull request closed without merging**